pkg/providers/instancetype/zz_generated.vpclimits.go

```diff
@@ -36,6 +36,8 @@ type VPCLimits struct {
 	BranchInterface         int
 	NetworkCards            []NetworkCard
 	DefaultNetworkCardIndex int
+	Hypervisor              string
```

Comment:
Can we do a quick analysis to see how they are using these new values? We were caught before with the change to the `DefaultNetworkCardIndex` when they started changing their logic into how they were calculating `maxPods` for certain instances. I wonder if the same is true here.

Reply:
Discussed offline. The VPC networking team is using these values to see which instance types can enable PrefixDelegation on Windows. This is unimportant for Karpenter right now so there are no core changes to the Karpenter max-pods logic based on these additions.